app/javascript/report_form_multiselect.vue

```diff
@@ -0,0 +1,438 @@
+<template lang="pug">
+div
+  multiselect(
+    v-model='selected',
+    :options='options',
+    placeholder='',
+    :multiple='true',
+    label='title',
+    track-by='title',
+    @select='select($event)',
+    @remove='remove($event)'
+  )
+  #multiselect__values
+</template>
+
+<script>
+import Multiselect from 'vue-multiselect'
+export default {
+  components: {
+    Multiselect
+  },
+  props: {
+    practices: { type: String, required: true },
+    editpractices: { type: [Array], default: () => [] }
+  },
+  data() {
+    const jsonPractices = JSON.parse(this.practices)
+    const practices = jsonPractices.map((practice) => {
+      const robj = { title: practice[0], id: practice[1] }
+      return robj
+    })
```

Comment:
変数に代入しなければ、以下のように短く書けるかも？

```suggestion
    const practices = jsonPractices.map((practice) => { title: practice[0], id: practice[1] })
```